src/platform/web_app_wayland.cc

```diff
@@ -194,6 +194,7 @@ void WebAppWayland::Init(int width,
     case AglShellSurfaceType::kPanel:
       app_window_->SetAglPanel(static_cast<int>(panel_type));
       surface_role_ = surface_role;
+      break;
```

Comment:
This most likely needs to be a distinct patch, as it clearly fixes a bug.

Reply:
Sure, I will add an independent fix for this one.